### ISSUE TITLE
TST: Use GitHub Actions

### ISSUE THREAD

**Issue by pllim** [astropy/montage-wrapper#65]
Please see https://groups.google.com/g/astropy-dev/c/45frnTEAX-U .

@astrofrog , are you able or willing to take this one? Please let me know either way. Thanks!

**Comment by astrofrog** [comment]
Yep! Though actually this package isn't really supported anymore. Will take a look.

**Comment by pllim** [comment]
Well, archiving it is also a solution...

**Comment by eteq** [comment]
In some out-of-band discussion we opted to go the archive route.  So I am archiving this repo as it stands. If anyone wants to consider bringing it back feel free to bring the topic up in astropy/project or other astropy discussion venues.
